packages/schematics/src/collection/node-application/files/app/proxy.conf.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
How does this file get used? wouldn't the node-builder have to start the proxy server or whatever the browser builder does?

I imagined this would be in the frontend application so each frontend can run a different port and point to the same backend.